content/en/logs/guide/azure-logging-guide.md

```diff
@@ -278,72 +278,46 @@ See [Diagnostic settings in Azure monitor][213] for more information.
+1. Enter the details of the storage account created by the deployment.
 
-See [Getting started with Azure Functions][307] for more information.
+Alternatively, you can view all resources that support diagnostic settings on the [**Monitor > Diagnostic settings**][305] page in the Azure portal.
 
-##### Point your Blob Storage trigger to Datadog
+#### Activity logs
 
-1. On the detail page of your Event Hub trigger function, click **Code + Test** under the **Developer** side menu.
-2. Add the [Datadog-Azure Function code][308] to the function's `index.js` file.
-3. Add your Datadog API key with a `DD_API_KEY` environment variable, or copy it into the function code by replacing `<DATADOG_API_KEY>` on line 20.  
-4. If you're not using the Datadog US1 site, set your [Datadog site][312] with a `DD_SITE` environment variable under the configuration tab of your function app, or copy the site parameter into the function code on line 21.
-5. **Save** the function.
-6. Click **Integration** under the **Developer** side menu.
-7. Click **Azure Blob Storage** under **Trigger and inputs**.
-8. Set the **Blob Parameter Name** to `blobContent` and click **Save**.
-9. Verify your setup is correct by checking the [Datadog Log Explorer][310] for logs from this resource.
-
-[301]: https://azure.microsoft.com/en-us/services/storage/blobs/
-[302]: https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal
-[303]: https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-storage-explorer
-[304]: https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-cli
-[305]: https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-powershell
-[306]: https://learn.microsoft.com/en-us/training/modules/store-app-data-with-azure-blob-storage/
-[307]: https://learn.microsoft.com/en-us/azure/azure-functions/functions-get-started
-[308]: https://github.com/DataDog/datadog-serverless-functions/blob/master/azure/blobs_logs_monitoring/index.js
-[309]: https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Web%2Fsites/kind/functionapp
-[310]: https://app.datadoghq.com/logs
-[311]: https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#configure-a-connection-string-for-an-azure-storage-account
-[312]: https://docs.datadoghq.com/getting_started/site/
-[313]: https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob-trigger?tabs=python-v2%2Cisolated-process%2Cnodejs-v4%2Cextensionv5&pivots=programming-language-csharp
+To [forward Activity logs][306]:
+1. In the Azure portal, go to the [**Monitor > Activity log**][307] page.
+1. Select **Export Activity Logs**.
+1. [Create a diagnostic setting][308] and select the categories you want to forward.
+1. Under **Destination details**, select **Archive to a storage account**.
+1. Enter the details of the storage account created by the deployment.
```

Comment:
*nit similar to above, we could provide some information on how to find the storage account